controller/role_list/index.js

```diff
@@ -0,0 +1,22 @@
+const con = require("../../connection");
+
+const handlerRoleList = (req, res) => {
+  console.log(req.params.role_id);
+  console.log(req.params.function_id);
+  try {
+    con.query(
+      `SELECT * FROM ROLE_LIST WHERE role_id = ${req.params.role_id} AND function_id = ${req.params.function_id};`,
```

Comment:
`SELECT * FROM ROLE_LIST WHERE role_id = ${role_id} AND function_id = ${function_id};`

Reply:
เรียกแบบนี้มัน error 500